files/scripts/swss.sh

```diff
@@ -339,6 +339,23 @@ start() {
         $SONIC_DB_CLI APPL_STATE_DB FLUSHDB
         clean_up_chassis_db_tables
         rm -rf /tmp/cache
+        if [[ x"$sonic_asic_platform" == x"broadcom" ]]; then
+            . /host/machine.conf
+            if [ -n "$aboot_platform" ]; then
+                platform=$aboot_platform
+            elif [ -n "$onie_platform" ]; then
+                platform=$onie_platform
+            else
+                platform="unknown"
+            fi
+            # Need to restart PMON on media_settings.json skus due to
+            # https://github.com/sonic-net/sonic-buildimage/issues/21902
+            if [[ x"$platform" == x"x86_64-arista_7800r3a"* ]]; then
```

Comment:
is this intended to be added in non-warmboot?

Reply:
This is intentional, if you look at the lines within that conditional it's only on non-warmboot that we flush `APPL_DB` which is what will delete the XCVR tuning values.

Reply:
@arista-nwolfe I don't see a need to have platform specific check. Can you please remove the platform check? Unless I am missing something or you can explain your concern.

Reply:
@arlakshm can you comment as per our offline discussion?

Reply:
Hi @arista-nwolfe, I had offline discussion with @prgeor, it will good that we make this generic to all platform with `media_settings.json` file, but as you pointed out it might be risky for 202405. So for the 202405 can you please port these changes to msft/202405 for the master branch we can make the change platform agnostic?

Reply:
Here is the msft-202405 PR: https://github.com/Azure/sonic-buildimage-msft/pull/1080
I'll close this PR for master